ush/set_rrfs_config_general.sh

```diff
@@ -1,7 +1,8 @@
 
 if [[ $MACHINE == "jet" ]] ; then
+  ACCOUNT=nrtrr
   if [[ -n $RESERVATION ]] ; then
-    ACCOUNT="${ACCOUNT:-rtrr}"
+    ACCOUNT=rtrr
```

Comment:
Why do we change from `ACCOUNT="${ACCOUNT:-rtrr}` to a hardwired one? The previous one will be more flexible.

Reply:
It can not get the right account if the configure set it wrong. This will use the right account for now. Need a better way to set the account.